ui/src/pages/version/components/forms/components/transformer/components/table_inputs/TableColumnsInput.js

```diff
@@ -0,0 +1,167 @@
+import React from "react";
+import { EuiButtonIcon, EuiFieldText, EuiSuperSelect } from "@elastic/eui";
+import { get, InMemoryTableForm, useOnChangeHandler } from "@gojek/mlp-ui";
+
+export const TableColumnsInput = ({
+  variables,
```

Comment:
`tables` is probably a better name

Reply:
It is not tables. It is columns actually. But "columns" is already used. Maybe I will rename is "cols"

Reply:
I see, I think "columns" is better, and we rename the existing "columns" to "cols" as it's internal constant.